src/main/java/uk/gov/hmcts/dts/fact/services/admin/list/AdminLocalAuthorityService.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.dts.fact.services.admin.list;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.dts.fact.exception.DuplicatedListItemException;
+import uk.gov.hmcts.dts.fact.exception.NotFoundException;
+import uk.gov.hmcts.dts.fact.model.admin.LocalAuthority;
+import uk.gov.hmcts.dts.fact.repositories.LocalAuthorityRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.stream.Collectors.toList;
+
+@Service
+public class AdminLocalAuthorityService {
+
+    private final LocalAuthorityRepository localAuthorityRepository;
+
+    @Autowired
+    public AdminLocalAuthorityService(final LocalAuthorityRepository localAuthorityRepository) {
+        this.localAuthorityRepository = localAuthorityRepository;
+    }
+
+    public List<LocalAuthority> getAllLocalAuthorities() {
```

Comment:
What is the difference between this method and Majid's method in AdminCourtLocalAuthoritiesService.class getAllLocalAuthorities(). They look the same no? Can we not combine them? I think we can remove Majid's one at some point as it is better to be placed under 'list' (not specific to a single court). Needs a thought as it needs to coordinate with the front end changes

Reply:
Agreed. I intend to raise a new ticket for moving the local authorities tab to use this new endpoint and remove the duplicate.